plugins/inputs/redis_sentinel/redis_sentinel_test.go

```diff
@@ -0,0 +1,382 @@
+		"role_reported_time":      "10267432",
+		"runid":                   "70e07dad9e450e2d35f1b75338e0a5341b59d710",
+		"slave_priority":          "100",
+		"slave_repl_offset":       "1392400",
+	}
+
+	var acc testutil.Accumulator
+
+	sentinelTags, sentinelFields := convertSentinelReplicaOutput(&acc, globalTags, masterName, replicasOutput)
+	actualMetrics := []telegraf.Metric{
+		testutil.MustMetric(measurementReplicas, sentinelTags, sentinelFields, now),
+	}
+
+	testutil.RequireMetricsEqual(t, expectedMetrics, actualMetrics)
+}
+
+func TestRedisSentinelInfoAll(t *testing.T) {
+	now := time.Now()
+
+	globalTags := map[string]string{
+		"port":   "6379",
+		"source": "redis.io",
+	}
+
+	expectedTags := map[string]string{
+		"port":   "6379",
+		"source": "redis.io",
+	}
+
+	expectedFields := map[string]interface{}{
+		"lru_clock":     int64(15585808),
+		"uptime_ns":     int64(901000000000),
+		"redis_version": "5.0.5",
+
+		"clients":                         int64(2),
+		"client_recent_max_input_buffer":  int64(2),
+		"client_recent_max_output_buffer": int64(0),
+		"blocked_clients":                 int64(0),
+
+		"used_cpu_sys":           float64(0.786872),
+		"used_cpu_user":          float64(0.939455),
+		"used_cpu_sys_children":  float64(0.000000),
+		"used_cpu_user_children": float64(0.000000),
+
+		"total_connections_received":     int64(2),
+		"total_commands_processed":       int64(6),
+		"instantaneous_ops_per_sec":      int64(0),
+		"total_net_input_bytes":          int64(124),
+		"total_net_output_bytes":         int64(10148),
+		"instantaneous_input_kbps":       float64(0.00),
+		"instantaneous_output_kbps":      float64(0.00),
+		"rejected_connections":           int64(0),
+		"sync_full":                      int64(0),
+		"sync_partial_ok":                int64(0),
+		"sync_partial_err":               int64(0),
+		"expired_keys":                   int64(0),
+		"expired_stale_perc":             float64(0.00),
+		"expired_time_cap_reached_count": int64(0),
+		"evicted_keys":                   int64(0),
+		"keyspace_hits":                  int64(0),
+		"keyspace_misses":                int64(0),
+		"pubsub_channels":                int64(0),
+		"pubsub_patterns":                int64(0),
+		"latest_fork_usec":               int64(0),
+		"migrate_cached_sockets":         int64(0),
+		"slave_expires_tracked_keys":     int64(0),
+		"active_defrag_hits":             int64(0),
+		"active_defrag_misses":           int64(0),
+		"active_defrag_key_hits":         int64(0),
+		"active_defrag_key_misses":       int64(0),
+
+		"sentinel_masters":                int64(2),
+		"sentinel_running_scripts":        int64(0),
+		"sentinel_scripts_queue_length":   int64(0),
+		"sentinel_simulate_failure_flags": int64(0),
+		"sentinel_tilt":                   int64(0),
+	}
+
+	expectedMetrics := []telegraf.Metric{
+		testutil.MustMetric(measurementSentinel, expectedTags, expectedFields, now),
+	}
+
+	const testINFOOutput = `
```

Comment:
Can you maybe move those to a file that is loaded to split code and data? Usually those files should reside in a `testdata` subdirectory...

Reply:
@srebhan Addressed in https://github.com/influxdata/telegraf/pull/10042/commits/e84bf2e360bfc70e76ff07ccc59f80e6bb19111f